Sprint-1/destructuring/exercise-3/exercise.js

```diff
@@ -6,3 +6,18 @@ let order = [
   { itemName: "Hot Coffee", quantity: 2, unitPricePence: 100 },
   { itemName: "Hash Brown", quantity: 4, unitPricePence: 40 },
 ];
+
+console.log("QTY     ITEM                TOTAL");
+
+let total = 0;
+
```

Comment:
It's best practice to declare variables that are not reassigned as 'const' instead of 'let'.
Can you review your variable declarations that are not reassigned and change them to const?

Here is a video you can watch: https://youtu.be/RE6qf3As-XU?si=_YaiEPeBC5Cx_j56